app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/options/PrivacyOptionsWidget.java

```diff
@@ -0,0 +1,199 @@
+
+            hide(REMOVE_WIDGET);
+        });
+
+        mTrackingSetting = findViewById(R.id.trackingProtectionButton);
+        mTrackingSetting.setChecked(SettingsStore.getInstance(getContext()).isTrackingProtectionEnabled());
+        mTrackingSetting.setOnCheckedChangeListener((compoundButton, enabled, apply) -> {
+            SettingsStore.getInstance(getContext()).setTrackingProtectionEnabled(enabled);
+            SessionStore.get().setTrackingEnabled(enabled);
+        });
+
+        mPermissionButtons = new ArrayList<>();
+        mPermissionButtons.add(Pair.create(findViewById(R.id.cameraPermissionButton), Manifest.permission.CAMERA));
+        mPermissionButtons.add(Pair.create(findViewById(R.id.microphonePermissionButton), Manifest.permission.RECORD_AUDIO));
+        mPermissionButtons.add(Pair.create(findViewById(R.id.locationPermissionButton), Manifest.permission.ACCESS_FINE_LOCATION));
+        mPermissionButtons.add(Pair.create(findViewById(R.id.storagePermissionButton), Manifest.permission.READ_EXTERNAL_STORAGE));
+
+        for (Pair<ButtonSetting, String> button: mPermissionButtons) {
+            if (mWidgetManager.isPermissionGranted(button.second)) {
+                button.first.setShowAsLabel(true);
+                button.first.setButtonText(getContext().getString(R.string.permission_enabled));
+            }
+            button.first.setOnClickListener(v -> {
+                togglePermission(button.first, button.second);
+            });
+        }
+
+
+        mResetButton = findViewById(R.id.resetButton);
+        mResetButton.setOnClickListener(v -> resetOptions());
+    }
+
+    @Override
+    protected void initializeWidgetPlacement(WidgetPlacement aPlacement) {
+        aPlacement.visible = false;
+        aPlacement.width =  WidgetPlacement.dpDimension(getContext(), R.dimen.developer_options_width);
+        aPlacement.height = WidgetPlacement.dpDimension(getContext(), R.dimen.developer_options_height);
+        aPlacement.parentAnchorX = 0.5f;
+        aPlacement.parentAnchorY = 0.5f;
+        aPlacement.anchorX = 0.5f;
+        aPlacement.anchorY = 0.5f;
+        aPlacement.translationY = WidgetPlacement.unitFromMeters(getContext(), R.dimen.restart_dialog_world_y);
+        aPlacement.translationZ = WidgetPlacement.unitFromMeters(getContext(), R.dimen.restart_dialog_world_z);
+    }
+
+    @Override
+    public void show() {
+        super.show();
+
+        mWidgetManager.addWorldClickListener(this);
+        mScrollbar.scrollTo(0, 0);
+    }
+
+    @Override
+    public void hide(@HideFlags int aHideFlags) {
+        super.hide(aHideFlags);
+
+        mWidgetManager.removeWorldClickListener(this);
+    }
+
+    private void togglePermission(ButtonSetting aButton, String aPermission) {
+        if (mWidgetManager.isPermissionGranted(aPermission)) {
+            showAlert(aButton.getDescription(), getContext().getString(R.string.security_options_permissions_reject_message));
```

Comment:
UI nit: can we alter the padding to be ~34px (or whatever that is in Android `dp` units) from each corner?

![image](https://user-images.githubusercontent.com/203725/54202330-0830bc80-448d-11e9-9b3f-1f0b730c77a6.png)

vs.

![image](https://user-images.githubusercontent.com/203725/54202416-331b1080-448d-11e9-9be6-2b726a83b991.png)